src/bundle/Resources/views/fieldtypes/preview/content_fields.html.twig

```diff
@@ -364,15 +364,31 @@
                 <td>{{ field.value.fileName }}</td>
             </tr>
             <tr>
-                <td>{{ 'ezimage.size'|trans|desc('Size') }}</td>
+                <td>{{ 'ezimage.size'|trans|desc('Size') }}:</td>
                 <td>{{ field.value.fileSize|ez_file_size( 1 ) }}</td>
             </tr>
             <tr>
-                <td>{{ 'ezimage.alternative_text'|trans|desc('Alternative text') }}</td>
+                <td>{{ 'ezimage.alternative_text'|trans|desc('Alternative text') }}:</td>
                 <td>{{ field.value.alternativeText }}</td>
             </tr>
+            <tr>
+                <td>{{ 'ezimage.master_dimensions'|trans|desc('Master dimensions') }}:</td>
+                <td>{{ field.value.width ~ 'px W' ~ ' - ' ~ field.value.height ~ 'px H' }}</td>
```

Comment:
Shouldn't it be translatable?

Reply:
That's good question. It seems rather universal, but on the other hand.
@SylvainGuittard

Reply:
I agree with Lukasz: full words and translatable.
thanks guys